modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java

```diff
@@ -91,9 +90,11 @@ public Map<String, Object> getParameters(
 		).build();
 	}
 
-	private int _getFirstDayOfWeek() {
+	private int _getFirstDayOfWeek(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
```

Comment:
How about changing this parameter to be just Locale? Do you agree?

Reply:
Done!